src/stores/pacbioLibraries.js

```diff
@@ -184,9 +179,6 @@ export const usePacbioLibrariesStore = defineStore('pacbioLibraries', {
      */
     async updateLibrary(libraryFields) {
       //Validate the libraryFields to ensure that all required fields are present
-      if (!validateFields(libraryFields)) {
-        return { success: false, errors: 'The library is invalid' }
-      }
```

Comment:
Is there any reason to  remove this? 
Would it better to have this validation, so we don't need to pass this to service in case the validation fails?

Reply:
We'd be validating the same thing twice. If it fails in the UI, it would also fail in the store (as far as I'm aware). Pools are also validated in the component, not the store, so there wasn't a lot of parity there.

Reply:
Sorry if it is not clear. 
To clarify my suggestion, what I meant is
- Validate the store object from the component using a store validation method.
- Populate errors and associate it with any invalid fields in that object.
- Display errors based on those fields in the component while editing.
- Check for these errors before sending a request to the service.
This approach ensures that we perform validation only once and check for existing errors, avoiding unnecessary service calls that would return the same errors